drivers/hmis/app/models/hmis/workflow_definition/task.rb

```diff
@@ -5,7 +5,9 @@
 # Tasks may include a swimline which can be used to determine which user is assigned to the corresponding step
 module Hmis::WorkflowDefinition
   class Task < Node
-    belongs_to :form_definition, class_name: 'Hmis::Form::Definition'
+    # Similar to Hmis::Form::Instance, Task has a form _identifier_, not a specific form, so that referral workflow steps can use newly published form versions
+    has_many :definitions, primary_key: :form_definition_identifier, foreign_key: :identifier, class_name: 'Hmis::Form::Definition'
```

Comment:
I liked calling this `form_definition[s]` rather than `definitions` to disambiguate from other kinds of definitions (ie workflow definition)